tests/lib/cli.js

```diff
@@ -1160,4 +1161,29 @@ describe("cli", () => {
         });
     });
 
+    describe("testing the cloned config", () => {
```

Comment:
It's still not exactly clear to me how this is testing this change. Do you mind explaining to me?

Reply:
here the fixture being tested is the same as the one mentioned in the issue. I tried to reproduce the same here. 

let me know if you want me to add it somewhere else.